packages/sdk/README.md

```diff
@@ -1,32 +1,53 @@
-# @buildwithsygma/sygma-sdk-core
+<div align="center">
+    <h1>Sygma SDK <img src="assets/logo.svg" alt="" height="26px"></h1>
+</div>
+<br>
+<p align="center">
+  <a href="https://github.com/sygmaprotocol/sygma-sdk/actions/workflows/ci.yaml">
+    <img src="https://github.com/sygmaprotocol/sygma-sdk/actions/workflows/ci.yaml/badge.svg" alt="Build Status">
+  </a>
+    <img alt="GitHub" src="https://img.shields.io/github/license/sygmaprotocol/sygma-sdk">
+  <a href="https://www.npmjs.com/package/@buildwithsygma/sygma-sdk-core">
+  <a href="https://discord.gg/ykXsJKfhgq">
+    <img alt="discord" src="https://img.shields.io/discord/593655374469660673?label=Discord&logo=discord&style=flat" />
+  </a>
+    <img alt="npm" src="https://img.shields.io/npm/v/@buildwithsygma/sygma-sdk-core">
+  </a>
+</p>
+<p align="center">
+  <a href="https://docs.buildwithsygma.com/">Sygma general documentation</a>
+  |
+  <a href="https://sygmaprotocol.github.io/sygma-sdk/">SygmaSDK typedocs</a>
+</p>
 
-Core primitives for bridging and message passing
+A Typescript SDK allowing users to easily add cross-chain capabilities such as bridging tokens and sending messages to their dApp projects.
 
-## Usage
+## Features
+
+Currently supported:
+
+- Transfering fungible assets between **Substrate** and **EVM** chains
```

Comment:
should be this a checkbox?

Reply:
WE will likely update this again, just copied the main repo readme.